data/strings/strings.txt

```diff
@@ -6014,6 +6014,88 @@
     zh-Hans = 指南针校准
     zh-Hant = 指南針校準
 
+  [compass_calibration_recommended]
+    comment = Toast text when compass calibration may improve the correctness of the current position arrow
+    tags = android
+    en = Improve arrow direction by moving the phone in a figure-eight motion to calibrate the compass.
```

Comment:
"figure-eight" is not something people know... it would be better to show a picture, even better a video. In the worst case please just add a link to some youtube video or an article.

Reply:
@rtsisyk This PR is an incremental improvement of what is already in the master. Unfortunately, Android Toasts are not clickable, and can't show any images. We can merge these changes now, and if you have time to implement them later with images and links, it would be even better.
I remember that iOS showed a full-screen system dialog asking to calibrate the compass when it was wrong. But in one of the iOS updates they changed it and I never saw that calibrate dialog again.

Reply:
If you have better text/wording ideas, please elaborate.

Reply:
Sky Map does this:
https://github.com/organicmaps/organicmaps/assets/72605561/79095958-1975-454d-973e-adf43cd7cdf0


https://github.com/sky-map-team/stardroid